packages/ethers/src/client.ts

```diff
@@ -755,8 +757,16 @@ export class Web3Modal extends Web3ModalScaffold {
     if (this.emailProvider) {
       this.emailProvider.onRpcRequest(request => {
         // We only open the modal if it's not a safe (auto-approve)
-        if (!W3mFrameHelpers.checkIfRequestIsAllowed(request)) {
-          super.open({ view: 'ApproveTransaction' })
+        if (W3mFrameHelpers.checkIfRequestExists(request)) {
+          if (!W3mFrameHelpers.checkIfRequestIsAllowed(request)) {
+            super.open({ view: 'ApproveTransaction' })
+          }
+        } else {
+          this.emailProvider?.rejectRpcRequest()
+          super.open()
+          setTimeout(() => {
+            SnackController.showError('This RPC method is not supported')
+          }, 300)
```

Comment:
So 
> so users don't end up with blank screen
is not correct right? They now see "This RPC Method is not supported"?

Reply:
If an RPC request is on our safe/not-safe list, we will handle it. Safe list requests will be automatically approved, and not-safe list requests will open the approve request view in the modal.

If an RPC request is not part of either of these lists, we will show "This RPC Method is not supported."

Reply:
Since this is a user facing notification, can we change it to something in terms of "Requested action not supported" and and console.log for developers where we say rpc not supported with logging the exact method that is not supported?